netbox/templates/account/profile.html

```diff
@@ -34,6 +34,10 @@ <h5 class="card-header">{% trans "Account Details" %}</h5>
               <th scope="row">{% trans "Account Created" %}</th>
               <td>{{ request.user.date_joined|annotated_date }}</td>
             </tr>
+            <tr>
+              <th scope="row">{% trans "Account Last Login" %}</th>
+              <td>{{ object.last_login|annotated_date }}</td>
```

Comment:
Account views are different from most object-based views, as they deal exclusively with the current user.
```suggestion
              <td>{{ request.user.last_login|annotated_date }}</td>
```